addons/sourcemod/scripting/vsh/bosses/boss_vagineer.sp

```diff
@@ -187,6 +187,7 @@ methodmap CVagineer < SaxtonHaleBase
 		
 		SetVariantInt(iSentryHealth);
 		AcceptEntityInput(iEntity, "SetHealth");	//Sets sentry health
+		SDK_RemoveObject(this.iClient, iEntity);	//Make the boss not the sentry's original owner so he gets to potentially build more of them
```

Comment:
Do we want to remove/destroy sentry when Vagineer class get `Destroy` mid-game? `RemoveObject` leaves sentry alive when engineer changes class/team, and not decaying health as vagineer is destroyed

Reply:
ok